chapters/fragops.txt

```diff
@@ -955,22 +956,24 @@ fragments generated by <<primsrast-polygons-basic,back-facing>>
 fragments generated by <<primsrast-polygons-basic,front-facing polygons>> or
 any other primitives.
 
-The comparison performed is based on the elink:VkCompareOp, compare mask
-[eq]#s~c~# , and stencil reference value [eq]#s~r~# of the relevant state
-set.
-The compare mask and stencil reference value are set by either the
+The comparison operation performed is determined by a elink:VkCompareOp
+value set by
+ifdef::VK_VERSION_1_3,VK_EXT_extended_dynamic_state[]
+flink:vkCmdSetDepthCompareOp::pname:depthCompareOp, or by
+endif::VK_VERSION_1_3,VK_EXT_extended_dynamic_state[]
+slink:VkStencilOpState::pname:compareOp during pipeline creation.
+
+The compare mask [eq]#s~c~# and stencil reference value [eq]#s~r~# of the
+relevant state set determine arguments of the comparison operation.
```

Comment:
This is C+P from the original text, it's meant to be referencing either the front or back facing state. I think this was clearer before because it was in the preceeding paragraph, whereas now they are separated.

Perhaps simply:

```suggestion
front or back stencil state set determine arguments of the comparison operation.
```

MIght also be worth reordering these paragraphs though.